protocol-configuration/proposalsConfig.ts

```diff
@@ -1,10 +1,12 @@
 import { ProposalCategory, TemplatedProposalDescription, TemplatedProposalsConfigMap } from '@custom-types/types';
 
+import fip_x from '@proposals/description/fip_x';
+
 const proposals: TemplatedProposalsConfigMap = {
   fuse_withdrawal_guard: {
     deploy: true, // deploy flag for whether to run deploy action during e2e tests or use mainnet state
     totalValue: 0, // amount of ETH to send to DAO execution
-    proposal: tip_117, // full proposal file, imported from '@proposals/description/fip_xx.ts'
+    proposal: fip_x as unknown as TemplatedProposalDescription, // temporary hacky workaround, this field isn't needed but can't be undefined
```

Comment:
Let's change the base type of this `proposal` var within the `TemplatedProposalsConfigMap` to include undefined as an acceptable type